env/production.env

```diff
@@ -5,7 +5,7 @@ NODE_ENV=production
 PORT=8081
 HOST=https://pwabuilder-cloudapk.azurewebsites.net
 
-JDK8PATH=/usr/lib/jvm/java-8-openjdk-amd64/
+JDK8PATH=/usr/lib/jvm/java-17-openjdk-amd64/
```

Comment:
"Since Java version 9, it has transitioned the default JKS keystore to PKCS12", so any new attempt to create a new signed key will fail for this jdk, there is already one issue related to this here: https://github.com/GoogleChromeLabs/bubblewrap/issues/693

I sent you some details in discord, maybe one quick workaround is to "ignore" one of the passwords and keep using a single one, example:

https://github.com/pwa-builder/CloudAPK/blob/8860909494368a067b71c1968f4533a6a2db71f4/build/bubbleWrapper.ts#L242C1-L247C1

ignore `storePassword` and pass `keyPassword` in both:

```js
      keypassword: signingInfo.keyPassword,
      password: signingInfo.keyPassword,
```

I'm not sure about the security concerns on this, but it definitely relates to bubblewrap code because you won't be able to pass `-storetype JKS`